x/auth/pubkey/types/expected_keepers.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// BankKeeper defines the expected interface contract the bank module requires
+type BankKeeper interface {
```

Comment:
Once we remove the fee payment as a bank transfer from the handler, we can remove this as well.